config/config.toml.example

```diff
@@ -0,0 +1,155 @@
+# A sample configuration for fedora-messaging. This file is in the TOML format.
+# For complete details on all configuration options, see the documentation
+# https://fedora-messaging.readthedocs.io/en/latest/configuration.html.
+
+amqp_url = "amqp://"
+
+publish_exchange = "amq.topic"
+
+callback = "hotness.consumers:BugzillaTicketFiler"
+
+# Note the double brackets below.
+# To add another binding, add another [[bindings]] section.
+[[bindings]]
+queue = "the-new-hotness"
+exchange = "amq.topic"
+routing_keys = [
+    "org.release-monitoring.prod.anitya.project.version.update",
+    "org.release-monitoring.prod.anitya.project.map.new",
+]
+
+[[bindings]]
+queue = "then-new-hotness"
+exchange = "amq.topic"
+routing_keys = [
+    "org.fedoraproject.prod.buildsys.task.state.change",
+]
```

Comment:
Having 2 `bindings` sections like that will work, but you may want to add the second routing key to the first list to get the same result.